core/os/android/adb/commands.go

```diff
@@ -513,11 +515,21 @@ func (b *binding) PrepareGpuProfiling(ctx context.Context, installedPackage *and
 		log.W(ctx, "Failed to query developer driver: %v, assuming no developer driver found.", err)
 	}
 
+	// Setup debug property.  The GPU driver may enable profiling only if the debug property is set.
```

Comment:
Nit: "The GPU driver ..." -> "Some Qualcomm drivers ..."

Reply:
No. Others/future drivers may now rely on this as well.

Reply:
Just to clarify, this is now the our expectation of all GPU drivers behaviour. All GPU vendors should rely on this property to check whether commands must be instrumented.